GitUI/UserControls/RepoStateVisualiser.cs

```diff
@@ -29,20 +29,20 @@ internal sealed class RepoStateVisualiser : IRepoStateVisualiser
 
         public (Image, Brush) Invoke(IReadOnlyList<GitItemStatus> allChangedFiles)
         {
-            var stagedCount = 0;
-            var unstagedSubmodulesCount = 0;
+            var indexCount = 0;
+            var workTreeSubmodulesCount = 0;
             var notTrackedCount = 0;
 
             foreach (var status in allChangedFiles)
             {
                 if (status.Staged == StagedStatus.Index)
                 {
-                    stagedCount++;
+                    indexCount++;
                 }
 
                 if (status.Staged == StagedStatus.WorkTree && status.IsSubmodule)
                 {
-                    unstagedSubmodulesCount++;
+                    workTreeSubmodulesCount++;
```

Comment:
I find this case a bit harder to understand. `unstagedSubmodule` makes it clear what's the adjective and what's the noun. `workTreeSubmodule` sounds like one noun. Is it the work tree of the submodule? Maybe `workTreeSubmoduleChangeCount`? Same for `indexCount`. There is only one index. `indexChangeCount`?

Reply:
workTreeChangeSubmoduleCount and indexChangeCount